R/reporter-junit.R

```diff
@@ -137,7 +137,7 @@ JunitReporter <- R6::R6Class("JunitReporter",
       } else if (inherits(self$out, "connection")) {
         file <- tempfile()
         xml2::write_xml(self$doc, file, format = TRUE)
-        write_lines(read_lines(file), self$out)
+        self$cat_line(brio::read_lines(file))
```

Comment:
This should still go to `self$out`, no?

Reply:
Thanks for catching this!

Reply:
Oh actually no, because `self$cat_line()` sends it to the right place. Phew!

Reply:
oh, ok.